src/client/components/StudyGroupLogo/StudyGroupLogo.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './StudyGroupLogo.styles.css';
+
+export const StudyGroupLogo = (props) => {
+  return (
+    <i>
+      <img src={props.path} width="50" alt="logo" />
+    </i>
+  );
+};
+
+StudyGroupLogo.propTypes = {
+  path: PropTypes.arrayOf(PropTypes.string),
```

Comment:
This should be `PropTypes.string`, not an array.

Reply:
it is giving me an error if I use PropTypes.string, because the prop (path) using files knob and it returns string array

<img width="960" alt="Untitled" src="https://user-images.githubusercontent.com/60918335/99477952-d8210880-2953-11eb-9db9-c99a0bf7d8db.png">

Reply:
Then this is the problem of storybook - fix the problem with arrays there, not in the component since the typical usage of the component will be with a single path, not an array.

Reply:
fixed and changed array to string